### PR TITLE
Add new app level warning message

### PR DESCRIPTION
**Pull request by AllForNothing**
1. Show an app-level warning if there is a stuck job
    
image

2. Chang `Replication finished` to `Replication status changed`

image

Thank you for contributing to Harbor!

# Comprehensive Summary of your change

# Issue being fixed
Fixes #(issue)

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [x] Made sure tests are passing and test coverage is added if needed.
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Comment by AllForNothing** [comment]
@OrlinVasilev Please help to review the PR. 
I'm not sure if the expression `Some jobs have been waiting for execution for more than 24 hours. Please check the job service dashboard. For more details, please refer to the Wike.` is ok

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 1]
```diff
@@ -1753,7 +1753,11 @@
         "SCHEDULE_PAUSE_BTN_INFO": "PAUSE — Met en pause toutes les programmations de tâches.",
         "SCHEDULE_RESUME_BTN_INFO": "REPRENDRE — Reprend les files d'attente de tâches à exécuter.",
         "WORKER_FREE_BTN_INFO": "Arrête les tâches en cours pour libérer le worker",
-        "CRON": "Cron"
+        "CRON": "Cron",
+        "WAITING_TOO_LONG_1": "Some jobs have been waiting for execution for more than 24 hours. Please check the job service ",
```
Comment (OrlinVasilev) [inline src/portal/src/i18n/lang/fr-fr-lang.json:1757]:
```suggestion
        "WAITING_TOO_LONG_1": "Certain jobs have been pending for execution for over 24 hours. Please check the job service ",
```

File: src/portal/src/i18n/lang/tr-tr-lang.json [hunk 2]
```diff
@@ -1786,7 +1786,11 @@
         "SCHEDULE_PAUSE_BTN_INFO": "PAUSE — Pause all schedules to execute.",
         "SCHEDULE_RESUME_BTN_INFO": "RESUME — Resume all schedule to execute.",
         "WORKER_FREE_BTN_INFO": "Stop the current running job to free the worker",
-        "CRON": "Cron"
+        "CRON": "Cron",
+        "WAITING_TOO_LONG_1": "Some jobs have been waiting for execution for more than 24 hours. Please check the job service ",
```
Comment (OrlinVasilev) [inline src/portal/src/i18n/lang/tr-tr-lang.json:1790]:
```suggestion
        "WAITING_TOO_LONG_1": "Certain jobs have been pending for execution for over 24 hours. Please check the job service ",
```

File: src/portal/src/i18n/lang/es-es-lang.json [hunk 3]
```diff
@@ -1783,7 +1783,11 @@
         "SCHEDULE_PAUSE_BTN_INFO": "PAUSE — Pause all schedules to execute.",
         "SCHEDULE_RESUME_BTN_INFO": "RESUME — Resume all schedules to execute.",
         "WORKER_FREE_BTN_INFO": "Stop the current running job to free the worker",
-        "CRON": "Cron"
+        "CRON": "Cron",
+        "WAITING_TOO_LONG_1": "Some jobs have been waiting for execution for more than 24 hours. Please check the job service ",
```
Comment (OrlinVasilev) [inline src/portal/src/i18n/lang/es-es-lang.json:1787]:
```suggestion
        "WAITING_TOO_LONG_1": "Certain jobs have been pending for execution for over 24 hours. Please check the job service ",
```

**Review by OrlinVasilev — COMMENTED** [review]
please check the new message

**Comment by AllForNothing** [comment]
@OrlinVasilev Thanks. PR updated

**Comment by OrlinVasilev** [comment]
@AllForNothing I see you have implemented my suggestions but you have force pushed over :) 
Super happy to help!

**Review by OrlinVasilev — APPROVED** [review]

**Review by wy65701436 — APPROVED** [review]
lgtm

**Pull request merged**